lib/issues.go

```diff
@@ -37,15 +37,18 @@ func CompareIssues(config cfg.Config, ghClient clients.GitHubClient, jiraClient
 		ids[i] = v.GetID()
```

Comment:
Due do this patch, this slice is never used. We should eliminate this loop and then this PR is good.